config/flags/flags.go

```diff
@@ -356,24 +356,29 @@ var (
 
 	// Event Emission Throttling
 	EnableThrottlingFlag = cli.BoolFlag{
-		Name:  "emitter.throttle-events",
-		Usage: "Enable event emission throttling based on network conditions",
+		Name:  "event-throttler.enable",
+		Usage: "Enable event emission throttling based on network conditions.",
 	}
 	ThrottlingDominantThresholdFlag = cli.Float64Flag{
-		Name:  "emitter.throttle-dominant-threshold",
-		Usage: "Dominant set stake threshold, driving event emission throttling.(percentage between 0 and 1)",
-		Value: emitter_config.DefaultConfig().ThrottlerDominantThreshold,
+		Name:  "event-throttler.dominant-threshold",
+		Usage: "The percentage of total stake which defines the dominating validator set. This value must be between 0.7 and 1.0.",
+		Value: emitter_config.DefaultConfig().ThrottlerConfig.DominantSetThreshold,
 	}
-	ThrottlingSkipInSameFrameFlag = cli.UintFlag{
-		Name:  "emitter.throttle-skip-in-same-frame",
-		Usage: "Maximum number of skipped events within the same frame to force event emission",
-		Value: emitter_config.DefaultConfig().ThrottlerSkipInSameFrame,
+	ThrottlingStalledFrameTimeout = cli.Uint64Flag{
```

Comment:
The unit of this parameter is `Attempts`, not `Frames`. Also, the term `Stalled` may also refer to the stopping of the emission of events in case of a disconnect to the network.

How about renaming this to `ThrottlingDominatingEmissionTimeout` to match the flag name?